docs/tutorials/smart-contract/cameligo.mdx

```diff
@@ -189,6 +189,12 @@ type returnValue = operation list * storage
 
 Before you can deploy the contract to Tezos, you must compile it to Michelson, the base language of Tezos contracts.
 
+1. Set the compiler to compile the namespace in your code:
```

Comment:
Namespace is module in cameligo. 

Also I would say "to target the module to compile in your code"